.travis.yml

```diff
@@ -1,4 +1,3 @@
-sudo: required
```

Comment:
Not related to main change. TravisCI no longer have two environments, so this line is no longer required.